ethminer/MinerAux.h

```diff
@@ -690,6 +729,9 @@ class MinerCLI
 			<< "        Use at your own risk! If GPU generates errored results they WILL be forwarded to the pool" << endl
 			<< "        Not recommended at high overclock." << endl
 #endif
+			<< " Temperature management: (implies -HWMON=0|1)" << endl
```

Comment:
Description should be set to

`"Temperature management: (implies -HWMON=1)"`

Would not work on without hwmon

Reply:
-HWMON=0 will be automatically enabled (m_show_hwmonitors=true which is what -HWMON=0 does).